docs/sources/api-response-examples/salesforce/sanitized/accounts_by_query.json

```diff
@@ -3,6 +3,9 @@
   "done":true,
   "records":[
     {
+      "attributes":{
+        "type":"Account"
```

Comment:
Sanitization produces same results than we had before, with the difference that the `attributes` object is present -required for doing the json validation for transformation-. They were redacted in the past not for security stuff, just because they are not being used in the caller.